NOTICES

```diff
@@ -40,7 +64,7 @@ TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
    control with that entity. For the purposes of this definition,
    "control" means (i) the power, direct or indirect, to cause the
    direction or management of such entity, whether by contract or
-   otherwise, or (ii) ownership of fifty percent (50%) or more of the
+   otherwise, or (ii) ownership of fifty percent (50%%) or more of the
```

Comment:
Is the second %% intentional?

Reply:
Don't know, but I copied the contents of the NOTICES file from what the PTC team sent back after doing the OSS scan.